### ISSUE TITLE
Typo in the docs

### ISSUE THREAD

**Issue by googol7** [reactabular/treetabular#16]
`retain`ing

instead of

`retaining`

here:

https://github.com/reactabular/treetabular#drag-and-drop

**Comment by bebraw** [comment]
I recall I wrote it that way on purpose to refer to the field exactly but either way is good to me.

**Comment by googol7** [comment]
alright i see. a little bit of css brushup would be good for the docs. the visual style doesn't look very appealing. in this case the font should be on the same baseline i think.

**Comment by bebraw** [comment]
Yeah, feel free to PR small tweaks. I agree the visual style isn't perfect in this case. Better fix the text I guess.
